### PR TITLE
Unregister component: front-end

### PR DESCRIPTION
**Pull request by Nek**
## Hey, I just made a Pull Request!

Implements the logic for showing a list of component from one location in the component unregister dialog. It doesn't work yet. We should make a decision on where to put the orchestration of a component(s) unregistration. I'd like to have it on backend in a transaction.

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

image

### CODE REVIEW

File: plugins/catalog/src/data/utils.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { Entity, Location } from '@backstage/catalog-model';
 import Edit from '@material-ui/icons/Edit';
 import IconButton from '@material-ui/core/IconButton';
 import { styled } from '@material-ui/core/styles';
+import { LOCATION_ANNOTATION } from '../../../../packages/catalog-model/src/location/annotation';
```
Comment (freben) [inline plugins/catalog/src/data/utils.tsx:22]: gotta update this to a real package import

**Comment by Nek** [comment]
@freben 👀

**Review by freben — APPROVED** [review]
Ok for now, let's get it in. But my main gripe with this is that we still have the entityToComponent concept here and writing more code that fits the old component model. It should all be completely torn out and replaced with direct use of plain Entity as-is. Maybe in a later PR?

**Comment by Nek** [comment]
> Ok for now, let's get it in. But my main gripe with this is that we still have the entityToComponent concept here and writing more code that fits the old component model. It should all be completely torn out and replaced with direct use of plain Entity as-is. Maybe in a later PR?

I totally agree. «Entia non sunt multiplicanda praeter necessitatem» I'll keep it in mind for the next PR.

**Pull request merged**